lib/controllers/project.js

```diff
@@ -320,12 +319,7 @@ Project.prototype._deploy = function (projectId, file, options, callback) {
     if (process.stdout.isTTY) { ubar.tick((progress * 100) - ubar.curr); }
   });
 
-  client.on('error', function(err) {
-    //closes the ubar readline interface
-    ubar.rl.close();
-
-    callback(err);
-  });
+  client.on('error', callback);
```

Comment:
I've tried testing, but haven't been able to see the `ubar` *NOT* close. Since I haven't run into any problems 👍